webui/react/Makefile

```diff
@@ -1,5 +1,7 @@
 swagger_client = "src/services/swagger"
-PROJECT_ROOT := "../../"
+export PROJECT_ROOT := "../../"
+# eg ./ for serving from root
+export PUBLIC_URL ?= /det
```

Comment:
**discussion:** Adding a note about having talked about changing line ~41 to be `SERVER_ADDRESS=http://localhost:8080 npm start` for the `make live` target.

Reply:
sg I'll just set it up in a way that can be overridden from outside if need be